utils/api/properties.js

```diff
@@ -0,0 +1,21 @@
+import axios from 'axios'
+
+console.log(process.env.ENDPOINT_API)
```

Comment:
Can we ✂️ this?

Reply:
Ah might have been left in an older commit. Should be removed in the most up to date commit but thanks for flagging

Reply:
Ah of course, yep it is. Thanks